cmake/templates/license.h.in

```diff
@@ -112,6 +112,12 @@ evmc:
    See the License for the specific language governing permissions and
    limitations under the License.
 
+docs/_static/img/share-solid.svg
```

Comment:
This is only added to the binary - maybe we should also just add a section to the documentation?

Reply:
You mean for all licenses or just for this? Seems like an overkill if it would be just for this one icon.

Reply:
I mean the icon is not present in the binary, why should he binary contain a copyright notice about the icon?

Reply:
Ah, I get what you mean now. OK, I can create a new page for this but this will be the only thing there :)

Reply:
New page added. I called it "Credits and attribution" with a section about website icons, meant to be similar to https://creativecommons.org/website-icons. With a link to the page in the footer, just like they do it.